### DISCUSSION TITLE
Is Deepin-Screen-Recorder possible in Fedora 41? Can you help?

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#10902] [Q&A | 问答板块]
Hello Deepin DDE Team

I was able to install Deepin-Screen-Recorder on Fedora 41 using the openSuse TW .rpm package.
It was necessary to install some dependencies manually and I see that it only works on X11 with some deficiency of incomplete subtitles.

In Wayland mode, I see that the application does not open when clicking on the application icon, much less via the terminal.

Is there a way to improve this application so that it can be used in Wayland mode on Fedora 41?

The output I get when trying to open the application through the terminal is the following:

```
edsonsantos@fedora:~$ deepin-screen-recorder
QSocketNotifier: Can only be used with threads started with QThread
dtk.core.dsg: Can't getId from AM for the  9373 , because AM is unavailable.
2024-11-15, 22:42:34.094 [Warning] [  ] 无法获取多任务视图dbus接口！
2024-11-15, 22:42:34.094 [Info] [  ] 多任务视图是否打开:  否
``` 

Manually installed dependencies

byzanz-0.3.0+git20140619-5.12.x86_64.rpm
byzanz-lang-0.3.0+git20140619-5.12.noarch.rpm
libdframeworkdbus2-6.0.0-1.1.x86_64.rpm

OpenSuse TW application version deepin-screen-recorder.rpm used:

deepin-screen-recorder-5.11.6-3.1.x86_64.rpm
deepin-screen-recorder-lang-5.11.6-3.1.noarch.rpm

QT_QPA_PLATFORM=wayland

Screen in X11 mode missing nomenclatures in the application's appearance.

![Captura de tela de 2024-11-15 22-15-20](https://github.com/user-attachments/assets/01b945b2-829d-44f8-a655-4bcf0c41d234)

Can anyone help me with this issue?

Thanks!

**Comment by BLumia** [comment]
I'm not exactly sure if deepin-screen-recorder currently support Wayland since deepin officially only support x11 session at the moment and DDE's Wayland support is currently still WIP, and deepin's official Wayland support might probably land in deepin 25 (the current version is deepin 23).

I'd suggest you consider using another screenshot utility that officially supports Wayland for now.
